core/domain/collection_domain.py

```diff
@@ -1030,6 +1030,30 @@ def get_node(self, exploration_id):
                 return node
         return None
 
+    def sort_nodes(self):
+        """Arrange the nodes in topologically sorted order unless it's
+        impossible, otherwise returns the unordered nodes.
+        """
+        unsorted_nodes = copy.deepcopy(self.nodes)
+        orderd_skills_ids = set()
```

Comment:
"ordered" is misspelled.

I thought I suggested accumulated_skill_ids? Stuff in a set isn't ordered. Also, pluralize only at the end, i.e. skill_ids rather than skills_ids.